designs/administrator/olm/list-of-operators/list-of-operators.md

```diff
@@ -0,0 +1,57 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# List of Operators
+
+## All-projects view
+This is the Installed Operators page where “all-projects” has been selected from the project-selector. Info icons have been added next to the Managed Namespaces column to provide the user more information about what these column titles mean.The user can sort the column via the sort icon (which looks like an arrow). There are a few additions to the flow:
+
+In the Managed Namespaces column there are three different ways the Namespace information can appear. In the Managed Namespaces column there are three different ways the Namespace information can appear.
+
+  a. “My-project” is representative of a single Namespace.
+
+  b. “All Namespaces” is a text only link that is associated with the Namespaces that fall across all Namespaces.
+
+  c. Thirdly, there is a multi-project view. For example, the Portworx Operator falls across multiple projects but not all projects, so in this example the Managed Namespaces column will have a popover on click. The Namespace names should be listed underneath as seen in the mockup.
+
+![3scale-all-projects-view1](img/3scale-all-projects-view1.jpg)
+On the Managed Namespaces column, when the user clicks the the popover it should read “Operands in these Namespace(s) are managed by the Operator”
+
+![3scale-all-projects-view2](img/3scale-all-projects-view2.jpg)
+On this screen the Portworx Operator falls across multiple projects but not all projects, so in this example the Managed Namespace column will have a popover on click.
+
+![3scale-all-projects-view3](img/3scale-all-projects-view3.jpg)
```

Comment:
Should the namespaces in this popover be links? I'm fine without the added complexity, but it just caught my eye since I know we have a "Project Dashboard" we could link to :)

I guess the same question could be asked of the "All Namespaces" text...we could link to the Projects Overview if we want to allow the user to go there...

Reply:
PM really wanted this part of the flows to not be links, and the reason was to prevent sending folks off to other parts of the flow. I'm open minded to this, but I'm wondering if we would want to reopen this discussion seeing that we already have the design in implementation?

Reply:
If PM has a strong opinion on it let's stick with that :)

Reply:
I do like Liz's ideas here, maybe something that could be revisited in future release.

Reply:
Sounds good to me, and I can see that being really helpful. Thanks @itsptk and @lizsurette. I'll take note!

Reply:
Sounds good!! Just blame me in the future if PM is annoyed ;)